src/components/organisms/PaneManager/PaneManager.tsx

```diff
@@ -215,7 +215,7 @@ const PaneManager = () => {
             <StartProjectPane />
           </div>
           {Boolean(projects.length) && (
-            <div style={{flex: 1, borderLeft: `1px solid ${Colors.grey3}`}}>
+            <div style={{flex: 1, borderLeft: `1px solid ${Colors.grey3}`, maxWidth: '28vw'}}>
```

Comment:
Can we create a styled container ( div ) inside `RecentProjectsPane` and remove the inline styling from the div here?

Reply:
yeah of course

Reply:
Perfect 👍🏻